src/components/LayoutTree.tsx

```diff
@@ -476,9 +476,14 @@ const TreeNode = ({
     ? `left {durations.layoutNodeAnimationFast} ease-out,top {durations.layoutNodeAnimationFast} ease-out`
     : `left {durations.layoutNodeAnimation} ease-out,top {durations.layoutNodeAnimation} ease-out`
 
+  const parentId = path.length > 1 ? path[path.length - 2] : null
+
   return (
     <div
       aria-label='tree-node'
+      data-id={thoughtId}
+      data-parent-id={parentId}
+      data-depth={depth}
```

Comment:
It looks like `data-depth` and `data-parent-id` are not used. 

`data-id` may also be able to be removed if the sibling elements are selected by editable `aria-label` as mentioned above.